packages/cli/src/wizard/wizard.js

```diff
@@ -29,33 +29,46 @@ async function runNewProjectWizard(options) {
   const fallbackProjectName = path.basename(process.cwd());
   const [defaultProjectName = fallbackProjectName] = (repoSlug && repoSlug.match(/[^/]+$/)) || [];
 
-  const responses = await inquirer.prompt([
-    {
+  const prompts = [];
```

Comment:
You'll want to add a typedef here, I'm not sure exactly what the type will be but it'll look something like this


```suggestion
  /** @type {Array<import('inquirer').WhateverThePromptTypeIs>} */
  const prompts = [];
```

Reply:
sorry, TS noob here, how to find the correct type ?

Reply:
if you're using vscode you can ctrl/cmd+click on the `import('inquirer')` bit and poke around the type definitions. In this case we want the same thing that `prompt` accepts, which looks like `Array<import('inquirer').Question>`